src/test-helpers/constants.ts

```diff
@@ -27,4 +27,4 @@ export const MAX_H512 =
 	'0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
 export const MAX_H256 =
 	'0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff';
-export const MAX_H160 = '0xffffffffffffffffffffffffffffffffffffffffffffffff';
+export const MAX_H160 = '0xffffffffffffffffffffffffffffffffffffffff';
```

Comment:
This just changes the byte length from 24 -> 20 as that is required for a H160 type and polkadot-js util will no longer shorten the length for you but error with an `assert`.